docs/Makefile

```diff
@@ -3,11 +3,15 @@
 
 # You can set these variables from the command line.
 SPHINXOPTS    =
-SPHINXBUILD  := `sh -c '[ -e /usr/bin/sphinx-build-3 ] && echo sphinx-build-3 || echo sphinx-build'`
+SPHINXBUILD  := sphinx-build
 SPHINXPROJ    = Flatpak
 SOURCEDIR     = .
 BUILDDIR      = _build
 
+ifeq ($(shell which $(SPHINXBUILD) >/dev/null 2>&1; echo $$?), 1)
```

Comment:
i'll change it tomorrow.

Reply:
--query doesn't seem to be a thing in bash/zsh. Do you mean `-v`? It will require redirecting though.

--query is a fish thing it seems https://fishshell.com/docs/current/cmds/command.html

Reply:
changed to -v